examples/evaluation.py

```diff
@@ -0,0 +1,38 @@
+from obstacle_tower_env import ObstacleTowerEnv, ObstacleTowerEvaluation
+import sys
+import argparse
+import time
+import multiprocessing
+
+def run_episode(env):
+    done = False
+    reward = 0.0
+    
+    while not done:
+        action = env.action_space.sample()
+        obs, reward, done, info = env.step(action)
+    return reward
```

Comment:
This reward is just the last one received, not the return for the episode.

Reply:
Addressed it in a commit.

Reply:
🤦‍♂